index.html

```diff
@@ -185,6 +185,50 @@ <h2 class="Recent ">About-me</h2>
                 </div>
             </div>
         </section>
+        <section class="form">
+            <h1 class="about-contact">
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </h1>
```

Comment:
- [ ] Please check my feedback from [ the video](https://youtu.be/Dmm04Kyzpyg) I recorded for you